gpytorch/likelihoods/multitask_gaussian_likelihood.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+from gpytorch.functions import add_diag
+from gpytorch.likelihoods import GaussianLikelihood
+from gpytorch.lazy import DiagLazyVariable, KroneckerProductLazyVariable
+
+
+class MultitaskGaussianLikelihood(GaussianLikelihood):
+    def __init__(self, n_tasks, log_noise_prior=None, log_task_noises_prior=None):
```

Comment:
Great that we have separate noise levels for the tasks. But if we also have a global log_noise prior don't we end up with an identifiability problem? That is, by scaling the noise by `alpha` and each task_noise by `1/alpha` we get a continuum of solutions for the MLL MAP estimate. 

Can we just get rid of the noise prior? To keep the API simple, we could still allow to pass in a single prior and then apply that to each of the task noises internally. 

I guess more generally, I wonder whether this works with a WhiteNoiseKernel when we provide variances for the observations (each task may have different observed variances)?